chart/cron-connector/templates/policy.yaml

```diff
@@ -0,0 +1,20 @@
+
+{{- if .Values.iam.enabled }}
+
+apiVersion: iam.openfaas.com/v1
+kind: Policy
+metadata:
+  name: {{ template "connector.fullname" . }}
+  namespace: openfaas
+  labels:
+    {{- include "connector.labels" . | nindent 4 }}
+spec:
+  statement:
+  - sid: 1-fn-r
+    action:
+    - Function:Read
```

Comment:
This should be Function:List, perhaps also Function:Get?

Reply:
The required actions are:
- Function:List
- Namespace:List

This has been updated and tested.